/*
package com.faol.security.exceptions;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertThrows;

public class RuntimeExceptionTest {

    @Test
    void runtimeExceptionTest(){

        String message = "runtime exception test";

        Assertions.assertAll(
                () -> assertThrows( java.lang.RuntimeException.class, () -> {
                    throw new RuntimeException(message);
                } )
        );
    }
}
*/
